src/components/Table/components/cell.tsx

```diff
@@ -30,6 +31,8 @@ export type GridCellContent = {
   css?: Properties;
   /** Allows cell to reveal content when the user hovers over a row. Content must be wrapped in an element in order to be hidden. IE <div>{value}</div>*/
   revealOnRowHover?: true;
+  /** Tooltip to add to a cell. **Only Works on Headers** */
```

Comment:
Question: Is the "only works on headers" still true? I thought scanning the story I saw tooltip on non-header rows.

Reply:
You're right. It works everywhere. I was going to just start on the headers, but seemed just as easy to do it everywhere once I thought of using `display: contents` in Tooltip.

Reply:
Sweet!